docs/interactivity.md

```diff
@@ -6,217 +6,182 @@ This page shows the payloads for browser request/responses, and App server reque
 
 <details><summary>Request from Browser</summary>
 
-`GET` http://localhost:8065/plugins/com.mattermost.apps/api/v1/bindings?user_id=mum5qskypidf3x3enkindgajrh&channel_id=zanqhwfdtjfi8yqyapd5qh6udh&scope=webapp
+`GET` /plugins/com.mattermost.apps/api/v1/bindings?user_id=ws4o4macctyn5ko8uhkkxmgfur&channel_id=qphz13bzbf8c7j778tdnaw3huc&scope=webapp
+
 </details>
 
 <details><summary>Response to Browser</summary>
 
 ```json
 [
     {
-        "app_id": "hello",
         "location": "/channel_header",
         "bindings": [
             {
-                "app_id": "hello",
+                "app_id": "builtin",
                 "location": "send",
-                "presentation": "modal",
```

Comment:
Why is the `presentation` property removed here? That is how a modal is opened with a binding, as used in https://github.com/mattermost/mattermost-plugin-apps/pull/45 . Can we have doc updates in their own PR so it's more obvious for discussion?

Reply:
Granted that I didn't intend the docs PR to be merged yet, as it does not reflect the code that's in master. Having a long running PR for docs is probably not the way to go, but that is what I originally intended.

So I propose we have a docs PR per significant change, or have the docs change within the code change PR as you have done here.